source/frontend/src/components/common/form/ContactInput/__snapshots__/ContactInputView.test.tsx.snap

```diff
@@ -217,7 +217,7 @@ exports[`ContactInputView component > renders as expected 1`] = `
                   fill="none"
                 />
                 <path
-                  d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
+                  d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
```

Comment:
I see a lot of changes in snaps related to icons and I wonder if you have updated your **node_modules** - @eddherrera ? Devin updated react-icons a while back 
It doesn't make sense to see all these changes in your PR as you didn't update any of these screens - did you?

Reply:
I haven't updated node_modules, will do a CI

Reply:
updated